src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelectorCore.js

```diff
@@ -229,6 +234,27 @@ class DraftJSMentionSelector extends React.Component<Props, State> {
         return !!(activeMention && activeMention.mentionString && contacts.length);
     };
 
+    buildAccessibleAlert = (activeMention: Object) => {
+        const { isFocused } = this.state;
+        const { contacts } = this.props;
+        let alertElement = null;
+        const addAlert = activeMention && activeMention.mentionString?.length > 0 && isFocused;
+        const previousElement = document.querySelector('[data-testid="accessibility-alert"]');
+        if (previousElement) {
+            previousElement.remove();
+        }
+        if (addAlert) {
+            alertElement = document.createElement('span');
+            alertElement.setAttribute('class', 'accessibility-hidden');
+            alertElement.setAttribute('data-testid', 'accessibility-alert');
+            alertElement.setAttribute('role', 'alert');
+            alertElement.innerText = contacts.length > 0 ? `${contacts.length} users found` : 'No users found';
```

Comment:
DraftJsEditor is a generic editor. These alerts look more specific to [CommentForm](https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/activity-feed/comment-form/CommentForm.js).

Reply:
On a different note - can you internationalize this text?